occupancy_transformer:
  ros__parameters:
    map_x_size: 10.0
    map_y_size: 20.0
    map_resolution: 0.1
    fuzz_threshold: 0.02
    base_link_height: 0.1
    ride_height: 0.05
    car_height: 0.15
    base_link_frame: base_link
    point_cloud_topic: /zed/zed_node/point_cloud/cloud_registered
